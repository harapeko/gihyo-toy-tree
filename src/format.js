exports.format = (node) => {
  return `${node.name}\n${formatEach(node.children, "")}`;
};
