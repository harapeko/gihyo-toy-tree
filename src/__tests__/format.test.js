import { format } from "../format";

describe("format", () => {
  test("only oot", () => {
    expect(
      format({
        type: "directory",
        name: "root",
        children: [],
      })
    ).toMatchSnapshot();
  });
});
