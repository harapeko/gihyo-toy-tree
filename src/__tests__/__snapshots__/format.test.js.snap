// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`format only oot 1`] = `
"root
"
`;
